honeybee_energy/aperturetype.py

```diff
@@ -0,0 +1,45 @@
+# coding=utf-8
+"""Aperture Types."""
+from honeybee.aperturetype import _ApertureType
+from honeybee.typing import float_in_range
+
+
+class OperableWindow(_ApertureType):
+    __slots__ = ('_fraction_operable')
+
+    """Type for operable apertures that can be opened for natrual ventilation."""
+
+    def __init__(self, fraction_operable=0.5):
+        """Initialize operable ApertureType.
+
+        Args:
+            fraction_operable: A number between 0 and 1 for the fraction of the
+                Aperture area that is operable. Default: 0.5.
+        """
+        self.fraction_operable = fraction_operable
+
+    @property
+    def fraction_operable(self):
+        """Get or set the fraction of the aperture area that is operable."""
+        return self._fraction_operable
+
+    @fraction_operable.setter
+    def fraction_operable(self, value):
+        self._fraction_operable = float_in_range(
+            value, 0.0, 1.0, 'aperture fraction operabe')
+
+    def to_dict(self):
+        """ApertureType as a dictionary."""
+        ap_type_dict = {
```

Comment:
That is because 'name' is the exact same as 'type' for `FaceType` and `ApertureType`.  I just replaced the 'name' dictionary key across all of the `FaceTypes` and `ApertureTypes` in honeybee-core and honeybee-energy with 'type'. So this is no longer an issue.